plugins/dataframe-gradle-plugin/src/integrationTest/kotlin/org/jetbrains/dataframe/gradle/SchemaGeneratorPluginIntegrationTest.kt

```diff
@@ -372,6 +375,127 @@ class SchemaGeneratorPluginIntegrationTest : AbstractDataFramePluginIntegrationT
         result.task(":build")?.outcome shouldBe TaskOutcome.SUCCESS
     }
 
+    @Test
+    // TODO: test is broken
```

Comment:
Sounds like a dataframe version mismatch

Reply:
Sorry, what do you mean?

Reply:
I think the gradle plugin uses the bootstrap version of dataframe, which does not yet include your added functions (like `readSqlTable` etc.). I remember from working on OpenAPI and adding support to the Gradle plugin that I continuously had to publishToMavenLocal because the Gradle plugin cannot use the source files directly.

Reply:
@koperagen am I correct?